qiskit_machine_learning/algorithms/trainable_model.py

```diff
@@ -196,3 +198,38 @@ def _choose_initial_point(self) -> np.ndarray:
         elif self._initial_point is None:
             self._initial_point = np.random.rand(self._neural_network.num_weights)
         return self._initial_point
+
+    def get_objective(
```

Comment:
I think `get_objective` may be private, e.g. `_get_objective`.

Reply:
Rectified in https://github.com/Qiskit/qiskit-machine-learning/pull/151/commits/da9be27a354e830db4597e3ada21f6a593254ffc